### PR TITLE
Introduce docs section to CONTRIBUTING.md

### PR DESCRIPTION
**Pull request by theletterf**
This PR adds a note to our `CONTRIBUTING.md` template to clarify that prescriptive guidance is hosted on https://docs.splunk.com/Observability and how can contributors send feedback about it.

### CODE REVIEW

**Comment by theletterf** [comment]
@flands The link to the LICENSE file in CONTRIBUTING.md throws a link checker error due to the file missing in the folder. We can either ignore the error or remove the link or make it relative (../../LICENSE).

```
Errors in specification/templates/CONTRIBUTING.md
✗ file:///github/workspace/specification/templates/LICENSE (Invalid file URI: file:///github/workspace/specification/templates/LICENSE)
```

**Review by mateuszrzeszutek — APPROVED** [review]

File: specification/templates/CONTRIBUTING.md [hunk 1]
```diff
@@ -1,43 +1,51 @@
...
-See [SECURITY.md](SECURITY.md#reporting-security-issues) for detailed instructions.
+See [SECURITY.md](SECURITY.md#reporting-security-issues) for instructions.
+
+## Documentation
+
+The Splunk Observability documentation is hosted on https://docs.splunk.com/Observability,
+which contains all the prescriptive guidance for Splunk Observability products. 
+Prescriptive guidance consists of step-by-step instructions, conceptual material,
+and decision support for customers. Reference documentation and development 
+documentation is hosted on this repository.
```
Comment (pellared) [inline specification/templates/CONTRIBUTING.md:40]: should it not be in the main `README.md` ?

File: specification/templates/CONTRIBUTING.md [hunk 2]
```diff
@@ -72,7 +80,7 @@ any 'help wanted' issues is a great place to start.
 
 ## Licensing
 
-See the [LICENSE](LICENSE) file for our repository's licensing. We will ask you to
+See the LICENSE file for our repository's licensing. We will ask you to
```
Comment (pellared) [inline specification/templates/CONTRIBUTING.md:83]: Can you revert this change? This PR addresses it https://github.com/signalfx/gdi-specification/pull/128

File: specification/repository.md [hunk 3]
```diff
@@ -107,6 +107,7 @@ approval is granted, GDI repositories MUST NOT cut a GA release.
   - SHOULD have badges on the `README.md` for other relevant things including artifacts
   - MUST have getting started information in `README.md`
   - MUST have troubleshooting information in `README.md`
+  - MUST link to official Splunk docs if they exist
```
Comment (pellared) [inline specification/repository.md:110]: other bullets repeat `README.md` all the time. Do you think it should be removed? I would prefer to keep consistency.

File: specification/templates/CONTRIBUTING.md (hunk 1 above)
Comment (theletterf) [reply specification/templates/CONTRIBUTING.md:40]: Possibly, but we don't have a README template. :)

File: specification/templates/CONTRIBUTING.md (hunk 2 above)
Comment (theletterf) [reply specification/templates/CONTRIBUTING.md]: Done!

File: specification/repository.md (hunk 3 above)
Comment (theletterf) [reply specification/repository.md:110]: Fixed!

**Review by pellared — APPROVED** [review]

**Comment by theletterf** [comment]
Guess we can merge? @flands

**Review by MrAlias — APPROVED** [review]

**Comment by MrAlias** [comment]
@theletterf it looks like https://docs.splunk.com/Observability is timing out. Is this expected?

**Comment by theletterf** [comment]
@MrAlias There's been an outage. It's fixed now!
